guides/telemetry.md

```diff
@@ -220,6 +252,16 @@ interface provided by `Telemetry.Metrics`. They then
 aggregate the measurements (data) into metrics to provide
 meaningful information about your application.
 
+For example, if the following `Telemetry.Metrics.summary/2` call is added to the `metrics/0` function of your Telemetry Supervisor:
+
+```elixir
+summary("phoenix.endpoint.stop.duration",
+  unit: {:native, :millisecond}
+)
+```
+
+Then the reporter will attach a listener for the `"phoenix.endpoint.stop.duration"` Telemetry event and will respond to this event by calculating a summary metric with the given event metadata and reporting on that metric to the appropriate source.
```

Comment:
At this point it should be pretty clear that we're talking about Telemetry events, so we can drop the additional descriptor.

```suggestion
Then the reporter will attach a listener for the `"phoenix.endpoint.stop.duration"` event and will respond to this event by calculating a summary metric with the given event metadata and reporting on that metric to the appropriate source.
```